target-app/Onboarding/SignIn/Views/SignInViewController.swift

```diff
@@ -91,31 +112,34 @@ private extension SignInViewController {
             lineView,
             signUpButton
         ])
-        
-        view.addSubview(stackView)
-        
-        configureStackView()
 
         activateConstraints()
-        
     }
     
-    func configureStackView () {
-        stackView.axis = NSLayoutConstraint.Axis.vertical
-        stackView.distribution = UIStackView.Distribution.fill
-//        stackView.alignment = UIStackView.Alignment.fill
-        stackView.spacing = 20
-//        stackView.backgroundColor = .systemYellow
-        
-        // Constraints
-        stackView.translatesAutoresizingMaskIntoConstraints = false
-        stackView.centerVertically(with: view)
-//        stackView.topAnchor.constraint(equalTo: titleLabel.bottomAnchor, constant: 20).isActive = true
-        stackView.leadingAnchor.constraint(equalTo: view.safeAreaLayoutGuide.leadingAnchor, constant: 0).isActive = true
-        stackView.trailingAnchor.constraint(equalTo: view.safeAreaLayoutGuide.trailingAnchor, constant: -0).isActive = true
-//        stackView.bottomAnchor.constraint(equalTo: lineView.topAnchor, constant: -20).isActive = true
+    private func setupViews() {
+        view.addSubview(scrollView)
+        scrollView.addSubview(contentView)
+        contentView.addSubview(stackView)
     }
     
+    private func setupLayout() {
+        scrollView.topAnchor.constraint(equalTo: view.safeAreaLayoutGuide.topAnchor).isActive = true
```

Comment:
instead of doing .isActive = true for each one of these, you could:
`NSLayoutConstraint.activate(<array of constraints>)`

Reply:
done